src/debugger.fs

```diff
@@ -148,12 +149,16 @@ module Program =
         with ex ->
             Debugger.showError ["Unable to connect to the monitor, continuing w/o debugger"; ex.Message]
             program
+            
+    let inline withDebuggerName (name:string) (program : Program<'a,'model,'msg,'view>) : Program<'a,'model,'msg,'view> =
+            try
+                let deflater, inflater = getTransformers<'model>()
+                let connection = Debugger.connect<'msg> (Debugger.ViaExtension name)
+                withDebuggerUsing deflater inflater connection program
```

Comment:
All the helper functions call `withDebuggerUsing`. Maybe the name parameter should be there? To avoid a breaking change, could introduce a new fn `withNamedDebuggerUsing` with name parameter, move current `withDebuggerUsing` code there. Then change `withDebuggerUsing` body to just call `withNamedDebuggerUsing`.

Reply:
@et1975 Thoughts?

Reply:
This should also reduce the usage of the default name down to 1 place.

Reply:
Could it be `string option`? Looks like Thoth's default behavior is to [skip null fields](https://thoth-org.github.io/Thoth.Json/documentation/auto/json-representation.html#skip-null-fields). When `name` is `None`, it should emit the same json as before this change. Or we could default it to "Elmish" before connecting. Then there's no need for an extra Using fn. Name is controlled by higher level fns.

Reply:
Or maybe it should be `| ViaExtension` and `ViaNamedExtension of name: string`.

Reply:
>  no need to worry about default in other fns.

I was wondering about that, @OnurGumus is naming supported with remote debugger?

Reply:
I think not; but I am not sure.